be/src/util/simdutf8check.h

```diff
@@ -17,7 +17,11 @@
 
 #pragma once
 
+#ifdef __x86_64__
 #include <x86intrin.h>
+#elif defined (__aarch64__)
```

Comment:
we should give a specific macro for build on aarch64 with ksl library available. if the ksl library is not available, we could still build successful without the performance boost.